ruby-3.0.gemfile

```diff
@@ -53,8 +53,8 @@ gem 'webmock', '>= 3.10.0'
 gem 'webrick', '>= 1.7.0'
 
 group :check do
-  gem 'rbs', '~> 3.6', require: false
-  gem 'steep', '~> 1.7.0', require: false
+  gem 'rbs', '~> 3', require: false
```

Comment:
I think this should also be `~> 3.7`.

Reply:
I'll unlock it, tbh, since we commit the lock files and bump them ourselves when needed.